.asf.yaml

```diff
@@ -26,6 +26,7 @@ github:
     rebase:   true
   collaborators:
     - michael-hoke
+    - rthomas320
```

Comment:
I believe he doesn't use @rthomas320  I think he uses @rt320 mainly

Reply:
He plans on using rthomas320 for the effort.  Read the dev list.

Reply:
Gotcha okay yeah I didn't see that he was going to switch over